tests/test_sr.py

```diff
@@ -819,6 +821,660 @@ def test_finding_site(self):
         assert len(item.ContentSequence) == 0
 
 
+class TestSourceImageForSegmentation(unittest.TestCase):
+
+    def setUp(self):
+        super().setUp()
+        file_path = Path(__file__)
+        data_dir = file_path.parent.parent.joinpath('data')
+        self._src_dataset = dcmread(
+            str(data_dir.joinpath('test_files', 'ct_image.dcm'))
+        )
+        self._ref_frames = [0, 4, 5]
+
+    def test_basic(self):
+        src_image = SourceImageForSegmentation(
+            self._src_dataset.SOPClassUID,
+            self._src_dataset.SOPInstanceUID
+        )
+        assert len(src_image.ReferencedSOPSequence) == 1
+        assert (
+            src_image.ReferencedSOPSequence[0].ReferencedSOPClassUID ==
+            self._src_dataset.SOPClassUID
+        )
+        assert (
+            src_image.ReferencedSOPSequence[0].ReferencedSOPInstanceUID ==
+            self._src_dataset.SOPInstanceUID
+        )
+
+    def test_basic_frames(self):
```

Comment:
I would suggest naming the method `test_construction_with_frame_reference` instead for clarity.

Reply:
Addressed in 4bea49d